internal/kafka/internal/api/public/api/openapi.yaml

```diff
@@ -652,6 +652,78 @@ paths:
       security:
       - Bearer: []
       summary: Returns the list of supported regions of the supported cloud provider
+  /api/kafkas_mgmt/v1/instance_types/{cloud_id}/{cloud_region}:
```

Comment:
We call it cloud_provider everywhere else, let's stick to that instead of `cloud_id`

Reply:
Thanks @pb82 I'll make that change